packages/@okta/vuepress-site/docs/guides/build-sso-integration/main/index.md

```diff
@@ -73,31 +75,32 @@ This portion of the guide takes you through the steps required to test your inte
 
 ### Assign users
 
-First you must assign your integration to one or more test users in your org:
+First, you must assign your integration to one or more test users in your org:
 
 1. Click the **Assignments** tab.
 1. Click **Assign** and then select either **Assign to People** or **Assign to Groups**.
 1. Enter the appropriate people or groups that you want to have Single Sign-On into your application, and then click **Assign** for each.
-1. For any people that you add, verify the user-specific attributes, and then select **Save and Go Back**.
+1. Verify the user-specific attributes for any people that you add, and then select **Save and Go Back**.
 1. Click **Done**.
 
 ### Test Single Sign-On
 
-1. Sign out of your administrator account in your development org. Click **Sign out** in the upper-right corner of the Admin Console.
+1. Sign out of your Okta org. Click **Sign out** in the upper-right corner of the Admin Console.
 1. Sign in to the Okta End-User Dashboard as the regular user that was assigned the integration.
-1. In your dashboard, click the Okta tile for the integration and confirm that the user is signed in to your application.
+1. Click the Okta tile for the integration and confirm that the user is signed in to your application.
 
 <StackSnippet snippet="test" />
 
 ## Next steps
 
-* After you complete your testing and your integration is working as expected, you can start the submission process to have your integration included in the [Okta Integration Network](https://www.okta.com/okta-integration-network/) catalog.
-* Our [Submit an app integration](/docs/guides/submit-app) guide takes you through the steps required to submit your SSO integration through the OIN Manager.
+After you complete testing your integration, you can start the submission process to have your integration included in the [Okta Integration Network](https://www.okta.com/okta-integration-network/) catalog:
```

Comment:
and may replace "integration" with "application integration" in the 2 places in this line. Also optional.